### PR TITLE
Don't override rootViewController

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13256
affects: Parent, Student, Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by vanwagonet** [comment]
I had a tough time reproducing the issue, but the docs list a bunch of things the `rootViewController` var does that we don't want to override anyway.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1009404&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Core/Core/ActAsUser/ActAsUserWindow.swift [hunk 1]
```diff
@@ -48,21 +54,6 @@ public class ActAsUserWindow: UIWindow {
         fatalError("init(coder:) has not been implemented")
     }
 
-    public override var rootViewController: UIViewController? {
-        didSet {
```
Comment (cobbal) [inline Core/Core/ActAsUser/ActAsUserWindow.swift]: From what I can tell, this isn't really an override at all, and is just adding an extra observer. See https://gist.github.com/cobbal/5b93442e8b5487a119a9c4d9f9711975

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent, Student, Teacher

[MBL-13256](https://instructure.atlassian.net/browse/MBL-13256)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.25% | 93.25% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/ActAsUser/ActAsUserWindow.swift (hunk 1 above)
Comment (vanwagonet) [reply Core/Core/ActAsUser/ActAsUserWindow.swift]: Does it matter if the original var was implemented in ObjC? Can you still reproduce the issue with this change?

Comment (cobbal) [reply Core/Core/ActAsUser/ActAsUserWindow.swift]: Same behavior for ObjC. I can only reproduce it now on iOS 12. Maybe the new modal presentation style doesn't have this issue... which would be great

**Pull request merged**